wasm/wasm_main.go

```diff
@@ -54,5 +55,9 @@ func main() {
 	global := js.Global()
 	global.Set("grol", js.FuncOf(jsEval))
 	global.Set("grolVersion", js.ValueOf(grolVersion))
+	err := extensions.Init()
+	if err != nil {
+		log.Critf("Error initializing extensions: %v", err)
```

Comment:
First time seeing Crit in your code, how does it differ from Fatal?

I mean you are using Fatal in main.go when using extension.Init

Reply:
yeah I changed it to crit because in wasm it complains there is no os.exit (was fatalf earlier in the pr before I tested)

Reply:
Why not changing it in main.go then?

Reply:
because in main.go I want it to die if there is a bug with extensions

Reply:
But then won't wasm complain about it?

I might have missed something

Reply:
`grol/wasm/wasm_main.go` is the program (main) built for wasm, it's different from `grol/main.go` (because it creates JS apis etc)